### ISSUE TITLE
Can't clone

### ISSUE THREAD

**Issue by mottosso** [mgear-dev/mgear#121]
Looks like one of the submodules references a repo via SSH, and requires credentials to clone.

```bash
PS C:\github\mottosso> git clone https://github.com/mgear-dev/mgear4.git xgear --recursive
Cloning into 'xgear'...
remote: Enumerating objects: 3524, done.
remote: Counting objects: 100% (388/388), done.
remote: Compressing objects: 100% (224/224), done.
remote: Total 3524 (delta 176), reused 171 (delta 164), pack-reused 3136
Receiving objects: 100% (3524/3524), 16.68 MiB | 8.28 MiB/s, done.
Resolving deltas: 100% (2229/2229), done.
Submodule 'vendor/QJsonModel' (https://github.com/dridk/QJsonModel.git) registered for path 'vendor/QJsonModel'
Submodule 'vendor/Qt.py' (https://github.com/mottosso/Qt.py) registered for path 'vendor/Qt.py'
Submodule 'vendor/TwistSpline' (https://github.com/blurstudio/TwistSpline.git) registered for path 'vendor/TwistSpline'
Submodule 'vendor/cvwrap' (https://github.com/chadmv/cvwrap.git) registered for path 'vendor/cvwrap'
Submodule 'vendor/grim_IK' (https://github.com/kattkieru/grim_IK.git) registered for path 'vendor/grim_IK'
Submodule 'vendor/maya-math-nodes' (https://github.com/serguei-k/maya-math-nodes.git) registered for path 'vendor/maya-math-nodes'
Submodule 'vendor/weightDriver' (https://github.com/IngoClemens/weightDriver.git) registered for path 'vendor/weightDriver'
Cloning into 'C:/github/mottosso/xgear/vendor/QJsonModel'...
remote: Enumerating objects: 175, done.
remote: Counting objects: 100% (53/53), done.
remote: Compressing objects: 100% (14/14), done.
remote: Total 175 (delta 42), reused 39 (delta 39), pack-reused 122
Receiving objects: 100% (175/175), 137.68 KiB | 1.22 MiB/s, done.
Resolving deltas: 100% (92/92), done.
Cloning into 'C:/github/mottosso/xgear/vendor/Qt.py'...
remote: Enumerating objects: 2873, done.
remote: Counting objects: 100% (76/76), done.
remote: Compressing objects: 100% (44/44), done.
remote: Total 2873 (delta 43), reused 52 (delta 32), pack-reused 2797
Receiving objects: 100% (2873/2873), 786.82 KiB | 4.47 MiB/s, done.
Resolving deltas: 100% (1778/1778), done.
Cloning into 'C:/github/mottosso/xgear/vendor/TwistSpline'...
remote: Enumerating objects: 169, done.
remote: Counting objects: 100% (35/35), done.
remote: Compressing objects: 100% (21/21), done.
remote: Total 169 (delta 19), reused 25 (delta 14), pack-reused 134
Receiving objects: 100% (169/169), 140.11 KiB | 1.24 MiB/s, done.
Resolving deltas: 100% (93/93), done.
Cloning into 'C:/github/mottosso/xgear/vendor/cvwrap'...
remote: Enumerating objects: 371, done.
remote: Total 371 (delta 0), reused 0 (delta 0), pack-reused 371
Receiving objects: 100% (371/371), 171.42 KiB | 2.20 MiB/s, done.
Resolving deltas: 100% (256/256), done.
Cloning into 'C:/github/mottosso/xgear/vendor/grim_IK'...
remote: Enumerating objects: 108, done.
remote: Total 108 (delta 0), reused 0 (delta 0), pack-reused 108
Receiving objects: 100% (108/108), 174.93 KiB | 2.73 MiB/s, done.
Resolving deltas: 100% (50/50), done.
Cloning into 'C:/github/mottosso/xgear/vendor/maya-math-nodes'...
remote: Enumerating objects: 1188, done.
remote: Counting objects: 100% (59/59), done.
remote: Compressing objects: 100% (51/51), done.
remote: Total 1188 (delta 15), reused 26 (delta 5), pack-reused 1129
Receiving objects: 100% (1188/1188), 271.63 KiB | 2.26 MiB/s, done.
Resolving deltas: 100% (771/771), done.
Cloning into 'C:/github/mottosso/xgear/vendor/weightDriver'...
remote: Enumerating objects: 169, done.
remote: Counting objects: 100% (34/34), done.
remote: Compressing objects: 100% (27/27), done.
remote: Total 169 (delta 5), reused 20 (delta 1), pack-reused 135
Receiving objects: 100% (169/169), 4.98 MiB | 8.43 MiB/s, done.
Resolving deltas: 100% (44/44), done.
Submodule path 'vendor/QJsonModel': checked out 'c73ed6c97ab1337438715019fbe7dc40d947d4eb'
Submodule path 'vendor/Qt.py': checked out '242ec430ff390cf24051819802b9eec1e665d3b2'
Submodule path 'vendor/TwistSpline': checked out '33a868182a8ad43acdf79ffc6af74e2ae62d8ab8'
Submodule path 'vendor/cvwrap': checked out '51367b050704817cf3e72d4be018c182b4dd8dcb'
Submodule 'cgcmake' (git@github.com:chadmv/cgcmake.git) registered for path 'vendor/cvwrap/cgcmake'
Cloning into 'C:/github/mottosso/xgear/vendor/cvwrap/cgcmake'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:chadmv/cgcmake.git' into submodule path 'C:/github/mottosso/xgear/vendor/cvwrap/cgcmake' failed
Failed to clone 'cgcmake'. Retry scheduled
Cloning into 'C:/github/mottosso/xgear/vendor/cvwrap/cgcmake'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:chadmv/cgcmake.git' into submodule path 'C:/github/mottosso/xgear/vendor/cvwrap/cgcmake' failed
Failed to clone 'cgcmake' a second time, aborting
Submodule path 'vendor/grim_IK': checked out 'e895fcf25242c51edb846d8b55acb59c1555dbb2'
Submodule path 'vendor/maya-math-nodes': checked out '669ace5366356c6038ef63ba7d5574f18a583ae9'
Submodule path 'vendor/weightDriver': checked out 'b08a01695763bf816e3a3f78f6d37227b565a9fe'
Failed to recurse into submodule path 'vendor/cvwrap'
```

**Comment by miquelcampos** [comment]
ouch!  I will change that ASAP

Thanks for the heads up!

**Comment by miquelcampos** [comment]
Hi @mottosso 

It looks like the module with the issue is a submodule inside cvwrap from @chadmv  that is pointing to his cmake repo 😅 using ssh
https://github.com/chadmv/cvwrap/blob/master/.gitmodules

@chadmv do you mind checking this? I was trying to figure out how to send a PR on this but couldn't find a way. 🙏  My apologies for the inconvenience

**Comment by miquelcampos** [comment]
@mottosso  I have removed cvwrap and other submodules related with C++ plugins. Now should be possible to clone correctly. 

@chadmv Since Maya 2020 introduced the proximityWrap I am stopping to vendor your cvwrap from Maya 2022 and new versions.
I will keep it for older versions since I have already compiled it and for backward compatibility with old projects.
Thank you very much for your plugin, it has served well to many many people and productions 🙇 🙏
